pkg/module/go_get_fetcher.go

```diff
@@ -103,6 +97,21 @@ func (g *goGetFetcher) Fetch(ctx context.Context, mod, ver string) (*storage.Ver
 	return &storageVer, nil
 }
 
+func setupModRepo(fs afero.Fs, goPathRoot, mod, ver string) (string, error) {
+	const op errors.Op = "goGetFetcher.setupModRepo"
+	sourcePath := filepath.Join(goPathRoot, "src")
+	modPath := filepath.Join(sourcePath, getRepoDirName(mod, ver))
+	if err := fs.MkdirAll(modPath, os.ModeDir|os.ModePerm); err != nil {
+		return "", errors.E(op, err)
+	}
+
+	// setup the module with barebones stuff
+	if err := Dummy(fs, modPath); err != nil {
+		return "", errors.E(op, err)
+	}
+	return modPath, nil
+}
+
```

Comment:
looks like a little bit of overlap with this:

https://github.com/gomods/athens/blob/89d9dd26091fc334fedfd8aa04364c750e6fba45/pkg/module/go_get_fetcher.go#L56-L66

but I think it's ok to leave as-is. just a comment, nothing you need to do